packages/flutter/test/widgets/two_dimensional_viewport_test.dart

```diff
@@ -161,10 +161,18 @@ void main() {
         ));
         await tester.pumpAndSettle();
 
+        // In the tests below the number of RepaintBoundary widgets depends on:
```

Comment:
The viewport itself is a repaint boundary.

Reply:
`find.byType(RepaintBoundary)` only returns RepaintBoundary widgets.

Reply:
Ah right, thanks!